src/_locales/en/messages.json

```diff
@@ -75,31 +75,31 @@
     "message": "Miscellaneous"
   },
   "passff_prefs_mark_fillable": {
-    "message": "Mark fillable input fields with icon"
+    "message": "Mark fillable forms with an icon-shaped form menu"
```

Comment:
Even though I see the confusion that the word `field` is already used to describe the fields in password entries, I don't think it's correct to say we are marking the forms with icons. Furthermore, it's not the menu that is icon-shaped.

Reply:
I'm not fond of the `icon-shaped form menu` either. Indeed, the menu doesn't have a shape of icon :+1: